pybliometrics/scopus/serial_title.py

```diff
@@ -264,3 +273,53 @@ def _parse_list(d, metric):
         return sorted(set(values))
     except (KeyError, TypeError):
         return None
+
+
+def _get_all_cite_score_years(self, named_info_list, named_rank_list, data) -> Optional[List[NamedTuple]]:
+    """Auxiliary function to get all information contained in cite score 
+    information list for the `CITESCORE` view."""
+    data = data.get('citeScoreYearInfo', [])
+
+    new_data = []
+    # Iterate through years
+    for d in data:
+        citeScoreInfo = d.get('citeScoreInformationList', [])[
+            0].get('citeScoreInfo', [])[0]
+        # Iterate through subject ranks
+        subject_rank = [named_rank_list(subjectcode=int(subject['subjectCode']),
+                        rank=int(subject['rank']),
+                        percentile=int(subject['percentile']))
+                        for subject in citeScoreInfo['citeScoreSubjectRank']]
+        # Create named tuple with info
+        Citescoreinfolist_year = named_info_list(year=int(d['@year']),
+                                                 status=d['@status'],
+                                                 documentcount=int(citeScoreInfo['scholarlyOutput']),
+                                                 citationcount=int(citeScoreInfo['citationCount']),
+                                                 citescore=make_float_if_possible(citeScoreInfo['citeScore']),
+                                                 percentcited=int(citeScoreInfo['percentCited']),
+                                                 rank=subject_rank)
+        # Append new data
+        new_data.append(Citescoreinfolist_year)
+
+    return new_data or None
+
+
+def _get_two_cite_score_years(self, named_info_list, data) -> Optional[List[NamedTuple]]:
+    """Auxiliary function to get information contained in cite score information
+    list for the `STANDARD` and `ENHANCED` view.
+    """
+
+    def create_namedtuple(data, mode, named_info_list):
+        year = data.get(f'citeScore{mode}Year')
+        cite_score = data.get(f'citeScore{mode}')
+
+        # To be consistent with old verion
+        if (year is None) and (cite_score is None):
+            return None
+        else:
+            return named_info_list(year=int(year), citescore=float(cite_score))
+
+    current = create_namedtuple(data, 'CurrentMetric', named_info_list)
+    tracker = create_namedtuple(data, 'CurrentTracker', named_info_list)
+
```

Comment:
Now that I see this function in full I don't think it's necessary. It's a bit intransparent from the perspective of the parent function/property that calls this function. Can you please take the auxiliary function `create_namedtuple()` and put it back into the parent function/property and create both `current` and `tracker` inside line 48?